### DISCUSSION TITLE
how to make menu open on hover

### DISCUSSION THREAD

**Discussion by antosubash** [saadeghi/daisyui#2080] [Q&A] [answered]
in v2 it works like that but now we have to click to open and it stays open. how do I change that? looking for v2 behavior in v3 for menu.

**Comment by saadeghi** [comment, marked as answer]
in v3 the hover style was replaced by the `<details>` native HTML element because it provides better accessibility among different environments (desktop, mobile, touch devices, keyboard navigation, etc)  
I wouldn't suggest using hover for menu because clicking/tapping provides better user experience but if you want to do that, you can add/remove the `open` attribute from `<details>` tag using JS hover event.  
More info on `<details>` tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details
